modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -1,12 +1,17 @@
 require 'resolv'
 require 'win32/open3'
 
-module Proxy::Dns
-  class Dnscmd < Record
+module Proxy::Dns::Dnscmd
+  class Record < ::Proxy::Dns::Record
     include Proxy::Log
     include Proxy::Util
     attr_reader :resolver
 
+    def self.record(attrs = {})
+      new(attrs.merge(:server => ::Proxy::Dns::Dnscmd::Plugin.settings.dns_server,
+                      :ttl => ::Proxy::Dns::Plugin.settings.dns_ttl))
```

Comment:
The migration moves dns_ttl into the nsupdate providers, should it add it also to dnscmd, or keep it in the main dns config?